docs/tendermint-core/configuration.md

```diff
@@ -94,24 +99,29 @@ prof_laddr = ""
 # so the app can decide if we should keep the connection or not
 filter_peers = false
 
-##### advanced configuration options #####
 
-##### rpc server configuration options #####
+#######################################################################
+###                 Advanced Configuration Options                  ###
+#######################################################################
+
+#######################################################
+###       RPC Server Configuration Options          ###
+#######################################################
 [rpc]
 
 # TCP or UNIX socket address for the RPC server to listen on
-laddr = "tcp://0.0.0.0:26657"
+laddr = "tcp://127.0.0.1:26657"
 
 # A list of origins a cross-domain request can be executed from
 # Default value '[]' disables cors support
 # Use '["*"]' to allow any origin
 cors_allowed_origins = []
 
 # A list of methods the client is allowed to use with cross-domain requests
-cors_allowed_methods = ["HEAD", "GET", "POST"]
+cors_allowed_methods = ["HEAD", "GET", "POST", ]
```

Comment:
```suggestion
cors_allowed_methods = ["HEAD", "GET", "POST"]
```

?

Reply:
This is a copy of what is generated. It looks odd. I can open a pr to fix this in the toml